client/transactions/list/test/index.tsx

```diff
@@ -555,10 +555,12 @@ describe( 'Transactions list', () => {
 				)
 			).not.toBe( -1 ); // amount
 			expect(
-				getUnformattedAmount( displayFirstTransaction[ 3 ] ).indexOf(
-					csvFirstTransaction[ 4 ]
+				-Number( getUnformattedAmount( displayFirstTransaction[ 3 ] ) )
```

Comment:
In this particular test this is asserting that -Number('0.50') === Number('-0.5')